WinUIGallery/ControlPages/CustomTitlebarPage.xaml

```diff
@@ -34,8 +34,8 @@
                         User can set a top-level UIElement (defined as appTitlebar here) as titlebar for the window. The system titlebar disappears and the chosen uielement starts acting like the titlebar. <LineBreak></LineBreak>
                         The Background and Foreground Color dropdowns set the foreground and background of titlebar and caption buttons respectively.
                     </TextBlock>
-                        <StackPanel Orientation="Horizontal" HorizontalAlignment="Stretch" VerticalAlignment="Top" Spacing="10">
-                        <Button x:Name="customTitlebar" Click="defaultTitlebar_Click"></Button>
+                    <StackPanel Orientation="Horizontal" HorizontalAlignment="Stretch" VerticalAlignment="Top" Spacing="10">
+                        <Button x:Name="customTitlebar" Click="customTitlebar_Click"></Button>
```

Comment:
so fallback titlebar needs nullptr to be passed instead of actuall uielement. that is happening in this fucntion ?